packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -639,6 +704,82 @@ const processTemplate = (relFile, inputDir, outputDir, context) => {
     }
 }
 
+/**
+ * Process the Application Extensions into the application-extensions directory.
+ *
+ * @param appExtensions - An array of the Application Extension names.
```

Comment:
When you say names here do you mean package names?

Reply:
I originally removed the "@salesforce" namespace because it was causing issues when creating temporary folders.

I changed to use the namespace again but in the temporary folders.